tests/tizen_capi/unittest_tizen_capi.cpp

```diff
@@ -2000,51 +2000,83 @@ TEST (nnstreamer_capi_singleshot, failure_01_n)
   ml_tensors_info_create (&out_info);
 
   /* invalid file path */
-  status = ml_single_open (&single, "wrong_file_name", in_info, out_info,
+  status = ml_single_open (&single, "wrong_file_name", NULL, NULL,
```

Comment:
If you want to test with NULL in/out-info instead of incorrect in/out-info, please create additional test cases.

Reply:
Setting null is to ignore in/out info param error, this line only checks filepath.

Reply:
Do you mean to separate null-param cases and invalid-param cases?

Reply:
We need both null-param cases and invalid-meta cases.
You may separate them or have both of them in a GTEST case.